README.md

```diff
@@ -140,6 +140,10 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 
 * [rip](https://github.com/nivekuil/rip) - A safe and ergonomic alternative to rm
 
+#### rename
+
+* [rnr](https://github.com/ChuckDaniels87/rnr) - A command-line tool to batch rename files and directories
+
```

Comment:
Than you for your contribution :) 

Could you follow alphabet order? In this case you need to add rename subsection between `rbenv` and `rm`.

Reply:
@TaKO8Ki 
oops sorry, fixed.